driver/manifest/manifest.go

```diff
@@ -312,21 +320,61 @@ func Maintainers(open OpenFunc) ([]Maintainer, error) {
 	return list, nil
 }
 
-const sdkName = "gopkg.in/bblfsh/sdk"
+const (
+	sdkNameLegacy = "gopkg.in/bblfsh/sdk"
+	sdkName       = "github.com/bblfsh/sdk"
+)
 
 // maxVersionToml finds a maximal version of a Babelfish SDK in the list of the projects.
```

Comment:
at this point I had to consult the internet on the difference between [maximum vs maximal](https://www.quora.com/What-are-the-differences-between-“maximum”-and-“maximal”-Especially-in-scientific-papers)

Reply:
In this case either is fine. I would have used "maximum" myself, but this is also correct.